shell/client/demo-client.js

```diff
@@ -103,21 +88,17 @@ Router.map(function () {
       //
       // 5. Take them into this grain.
 
-      // calculate the appId
-
       // We copy the appId into the scope so the autorun function can access it.
       const appId = this.params.appId;
       let signingIn = false;
 
       // TODO(cleanup): Is there a better way to do this? Before multi-identity, we could use the
       //   `userCallback` option of `Accounts.callLoginMethod()`, but now that doesn't work because
       //   it fires too early.
-      //
-      // Note that we don't use Template.instance().autorun() because the template gets destroyed
-      // and recreated during account creation.
       let done = false;
-      const handle = Tracker.autorun(function () {
+      handle = Tracker.autorun(function () {
```

Comment:
It turns out that `handle` being non-local wasn't actually necessary after refactoring this route to use `onRun`.